docs/configuration.md

```diff
@@ -50,3 +50,48 @@ via `AGG_CONFIG` environment variable:
 ```
 export ACG_CONFIG="clowder_config.json"
 ```
+
+## Service Log configuration
+
+Service Log configuration is in section `[service-log]` in config file
+
+```
+[service_log]
+enabled = false
+offline_token = ""
+token_refreshment_url = "https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token"
+url = "https://api.openshift.com/api/service_logs/v1/cluster_logs/"
+timeout = "15s"
+likelihood_threshold = 0
+impact_threshold = 0
+severity_threshold = 0
+total_risk_threshold = 3
+event_filter = "totalRisk > totalRiskThreshold"
+```
+
+- `enabled` determines whether the notifications service sends messages to Service Log
+- `offline_token` is an offline token used to retrieve online token via OpenID Connect
+- `token_refreshment_url` is a URL of OpenID Connect API for retrieval of new online token
+- `timeout` is a time used as a timeout when sending requests to Service Log API
+- `likelihood_threshold`,`impact_threshold`, `severity_threshold` and `total_risk_threshold` are values which can be used in `event_filter` for filtering messages sent to Service Log
+- `event_filter` is a condition string used to determine which messages will be sent to Service Log
+
+Please note that for correct functionality of Service Log integration, `dependencies` configuration should be also present.
+
+## Dependencies configuration
+
+Service Log configuration is in section `[dependencies]` in config file
```

Comment:
is this sentence ok? this doesn't contain the SL configuration

Reply:
You are right, sorry. Will fix in a moment.

Reply:
Fixed